src/site/resources/.htaccess

```
@@ -8,13 +7,16 @@
 # "License"); you may not use this file except in compliance
 # with the License.  You may obtain a copy of the License at
 #
-#    http://www.apache.org/licenses/LICENSE-2.0
+#   http://www.apache.org/licenses/LICENSE-2.0
 #
 # Unless required by applicable law or agreed to in writing,
 # software distributed under the License is distributed on an
 # "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
-# ----------------------------------------------------------------------------
-./mvnw spotless:apply -pl '!log4j-bom' -T2C
+#
+RewriteEngine On
+RewriteRule "^(/log4j/[23].x)/log4j-(core|api)/apidocs(.*)" "$1/javadoc/$2$3" [R=permanent]
+RewriteRule "^(/log4j/[23].x)/manual/scala-api.html" "/log4j/scala" [R=permanent]
+RewriteRule "^(/log4j/[23].x)/release-notes/index.html" "$1/release-notes.html" [R=permanent]
```

```suggestion
RewriteEngine On
RewriteBase "/log4j/3.x/"
RewriteRule "^manual/scala-api(\.html)?$" "/log4j/scala/" [R=permanent]
RewriteRule "^release-notes/(index(\.html)?)?$" "release-notes" [R=permanent]
```

This seems to work on my local server.

Remark that according to Git history we never had a `/log4j/3.x/log4j-api/apidocs/` folder.

It is also pretty much a personal taste, but I prefer to use URLs that do not end in `.html`. Redirects to folders on the other way must terminate in `/` to spare the client an additional redirection.